notatrix/src/utils/index.ts

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+import * as _ from "underscore";
+
+import * as constants from "./constants";
+import * as errors from "./errors";
+import * as funcs from "./funcs";
+import * as regex from "./regex";
+
+module.exports = _.extend({re: regex}, errors, constants, funcs);
```

Comment:
This file is a bit weird.. preferably we would write this as
```ts
export * from "./constants";
export * from "./errors";
export * from "./funcs";
export * as re from "./regex";
```
but that would require us to change a bunch of our usage-sites (particularly in the tests).  I think I'm OK just leaving this as-is for now and we can revisit later on when more stuff is migrated.